<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentitySettingsDialog</class>
 <widget class="QDialog" name="IdentitySettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Identity settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="lblQuickPassTimeout">
     <property name="text">
      <string>QuickPass timeout (minutes):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="spnQuickPassLength">
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spnPasswordVerifySeconds">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="spnQuickPassTimeout">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblQuickPassLength">
     <property name="text">
      <string>QuickPass length:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPasswordVerifySeconds">
     <property name="text">
      <string>Password verification time (seconds):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="grpOptionFlags">
     <property name="title">
      <string>Option flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkCheckForUpdates">
        <property name="text">
         <string>Check for updates</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUpdateAutonomously">
        <property name="text">
         <string>Update autonomously</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRequestSqrlOnlyLogin">
        <property name="text">
         <string>Request &quot;SQRL only&quot; login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRequstNoSqrlBypass">
        <property name="text">
         <string>Request no SQRL bypass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkMitmWarning">
        <property name="text">
         <string>Warn of possible MITM attack</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkClearQuickPassOnSleep">
        <property name="text">
         <string>Clear QuickPass upon blanking, suspend, etc.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkClearQuickPassOnUserSwitch">
        <property name="text">
         <string>Clear QuickPass upon user switching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEnableQuickPassTimeout">
        <property name="text">
         <string>Enable QuickPass timeout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkWarnOnNonCps">
        <property name="text">
         <string>Warn of non-CPS authentication</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spnPasswordVerifySeconds</tabstop>
  <tabstop>spnQuickPassTimeout</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IdentitySettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IdentitySettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
